modules/post/multi/escalate/aws_create_iam_user.rb

```diff
@@ -27,7 +27,12 @@ def initialize(info = {})
         ],
         'References' => [
           [ 'URL', 'https://github.com/devsecops/bootcamp/raw/master/Week-6/slides/june-DSO-bootcamp-week-six-lesson-three.pdf' ]
-        ]
+        ],
+        'Notes' => {
```

Comment:
I guess this would be CONFIG_CHANGES if there's side effects on AWS

Reply:
Added